config/initializers/revision_components.rb

```diff
@@ -0,0 +1,7 @@
+Pageflow.after_configure do |config|
+  config.page_types.each do |page_type|
+    page_type.revision_components.each do |component|
+      config.revision_components.register(component)
+    end
+  end
+end
```

Comment:
[Page type help entry registration](https://github.com/codevise/pageflow/blob/master/config/initializers/help_entries.rb) had the same problem before, but now that we replicate the pattern it becomes more important: there is no easy way to test this code. We could consider moving all of this initializer code that applies configuration changes according to registered page types to a method in [`Pageflow::PageTypes`](https://github.com/codevise/pageflow/blob/master/lib/pageflow/page_types.rb) and call that from a single `after_configure` block in a single initializer. This would make testing easy and keep all of this logic in one place. This will come in handy, when we extract page types to the paged entry type engine at some point.